addons/mail/static/src/models/messaging_notification_handler/messaging_notification_handler.js

```diff
@@ -194,6 +198,34 @@ function factory(dependencies) {
             }
         }
 
+        /**
+         * @private
+         * @param {Object} payload
+         * @param {integer} payload.id
+         * @param {String} payload.new_name
+         */
+         _handleNotificationChanelRenamed ({ id, new_name: name }) {
+            const channel = this.messaging.models['mail.thread'].findFromIdentifyingData({
```

Comment:
I would prefer if this used `insert` instead of `findFromIdentifyingData`.
As `insert` will create it if it's missing, and `findFromIdentifyingData` will not (which means there should always be a condition to guarantee existence after using that method).